spec/models/tool_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Tool, type: :model do
```

Comment:
Rspec is the current object so you can just say describe, like below. Also model is the default so you don't have to write that either.